client/client.go

```diff
@@ -1467,7 +1474,8 @@ func (c *Client) setupConsulSyncer() error {
 			}
 			consulServices, _, err := consulCatalog.Service(nomadServerServiceName, consul.ServiceTagRPC, consulOpts)
 			if err != nil {
-				mErr.Errors = append(mErr.Errors, fmt.Errorf("unable to query service %+q from Consul datacenter %+q: %v", nomadServerServiceName, dc, err))
+				c.logger.Printf("[WARN] client.consul: unable to query service %+q from Consul dc %q: %v", nomadServerServiceName, dc, err)
```

Comment:
Create a common `msg := fmt.Sprintf("unable to query service %+q from Consul dc %q: %v", nomadServerServiceName, dc, err)` and use it in the log and error